build.gradle

```diff
@@ -0,0 +1,53 @@
+plugins {
+	id 'idea'
+
+	id 'io.spring.dependency-management' version '1.0.11.RELEASE' apply false
+	id 'org.springframework.boot' version '2.6.5' apply false
+	id 'com.diffplug.spotless' version '6.3.0' apply false
+	// ^^ for some reason, can't use spotless in multiple subprojects without this ^^
+	id 'com.google.cloud.tools.jib' version '3.1.4' apply false
+	// ^^ see https://github.com/GoogleContainerTools/jib/blob/master/docs/faq.md#i-am-seeing-method-not-found-or-class-not-found-errors-when-building
+  	id 'org.sonarqube' version '3.3' apply false
+	id 'io.freefair.lombok' version '6.4.1' apply false
+	id 'org.hidetake.swagger.generator' version '2.19.2' apply false
+}
+
+repositories { mavenCentral() }
+
+allprojects {
+	dependencyLocking {
+		lockAllConfigurations()
+	}
+}
+
+subprojects {
+	group = 'bio.terra'
+	version = '0.76.0-SNAPSHOT'
```

Comment:
Since you're using a root `build.gradle`, it wouldn't be hard to edit this value in place, but I would recommend putting it in a separate root `gradle.properties` file and using a variable to refer to it here. Something to think about when you get to adding build and bumper GitHub workflows.